Examples/package-info/Sources/package-info/main.swift

```diff
@@ -1,10 +1,23 @@
 import Workspace
 
-let swiftpm = SwiftPMHelper(rootPackage: localFileSystem.currentWorkingDirectory!)
-let workspace = swiftpm.createWorkspace()
+/// The path of the package.
+let package = localFileSystem.currentWorkingDirectory!
 
+/// Determine the directory that contains SwiftPM's PackageDescription libs.
+let libDir = AbsolutePath(#file).appending(RelativePath("../../../.build/.bootstrap/lib/swift/pm"))
+
+// Load package's manifest.
+let loader = ManifestLoader(manifestResources: UserManifestResources(libDir: libDir))
+let manifest = try loader.load(packagePath: package)
```

Comment:
By using default params for most things, it's easy to load manifests now. All you need is the package path and the location of SwiftPM's lib.

Reply:
We can obviously improve this further.